src/moj/components/messages/README.md

```diff
@@ -4,15 +4,15 @@
 
 ### Installation
 
-You will need to install the following code at the bottom of `server.js`, just above `module.exports = app;`
+You will need to install the following code at the bottom of `server.js`, just above `module.exports = app`
```

Comment:
What is server.js? It's not part of our code... 

I assume it is [this file](https://github.com/alphagov/govuk-prototype-kit/blob/main/server.js) from the prototype kit, but that's not obvious, and not everyone uses the prototype kit, so we probably need some clarification here.